DXRModules/DeusEx/Classes/DXRReplaceActors.uc

```diff
@@ -99,6 +102,22 @@ function ReplaceAllianceTrigger(#var(prefix)AllianceTrigger a)
     a.Destroy();
 }
 
+function ReplaceShakeTrigger(#var(prefix)ShakeTrigger a)
```

Comment:
Shake it, baby
![2576](https://user-images.githubusercontent.com/30947252/181909262-dc9f179f-09dc-4e4d-b6e0-b216ebc65b4b.png)